CHANGELOG.next.asciidoc

```diff
@@ -172,6 +172,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix wrong uptime reporting by system/uptime metricset under Windows. {pull}12915[12915]
 - Print errors that were being omitted in vSphere metricsets. {pull}12816[12816]
 - Fix redis key metricset dashboard references to index pattern. {pull}13303[13303]
+- Check if fields in DBInstance is nil in rds metricset. {pull}13294[13294] {issue}13037[13037]
```

Comment:
No need to add this changelog entry if the metricset hasn't been released yet.

Reply:
@jsoriano rds metricset is released in 7.3 hmmmm but in beta. In this case, is this changelog needed?

Reply:
Oh, then yes, sorry. And this will need backport to 7.3.